lighthouse-cli/test/smokehouse/perf/expectations.js

```diff
@@ -72,11 +72,11 @@ module.exports = [
       audits: {
         'resource-summary': {
           score: null,
-          displayValue: '11 requests • 164 KB',
```

Comment:
for anyone else scratching their head how in the world this was affected by the fixture change when `.corsfont` doesn't appear anywhere else in `load-things.html`, the CSS rule is defined over in `cors-fonts.css` :)

Reply:
my fault, it was sloppy to change it only a little and then draw a line in the sand for the budgets at that point :)

No memory of why I didn't change the classname itself from what's in `fonts.html`, just operating as a human fuzzer at the time, I think :)

Reply:
oh I'm still not clear why we need to use it at all no harm no foul I was just baffled how his text change could result in fonts getting requested 😆 

I think I was just slow on the uptake that this was modeled after another file